versions.txt

```diff
@@ -5,3 +5,4 @@ acid@3.1.0
 agar@5.4.0
 alloy@8.3.0
 mcagar@6.2.0
+robin@8.1.0
```

Comment:
Why is this a minor version bump? There's nothing about the API that's changed and instead we're just fixing types that should have been exposed originally as far as I can see.

- [ ] Remove minor version bump unless we have a good reason

Reply:
We're exporting a thing that wasn't previously exported — even if it maybe should have been exported before, that sounds like a minor bump to me.

Worth noting, this will probably get promoted to a major bump as part of the Optionals work in the other PR, so it's probably a moot point.

Reply:
As discussed via Slack, I disagree with that statement but do agree this will end up being a minor anyways. As such, I'm going to resolve this.